src/pages/Entries/ScrollableCalendar.tsx

```diff
@@ -28,13 +28,15 @@ export default function ScrollableCalendar({
   );
   const [madeFirstSelection, setMadeFirstSelection] = useState(false);
 
-  // start calendar at today if novalid date argument provided, otherwise start at the provided date
+  // start calendar at today if no valid date argument provided, otherwise start at the provided date
   const location = useLocation();
-  const startDateStr = location.state?.date ?? new URLSearchParams(useLocation().search).get("date");
+  const startDateStr =
+    location.state?.date ??
+    new URLSearchParams(useLocation().search).get("date");
   const today = modifyDateDay(new Date(), 0);
   const startDate =
     startDateStr && !isNaN(dateFromStr(startDateStr).getTime())
-      ? parseYYYYmmddFormat(startDateStr)
+      ? dateFromStr(startDateStr)
```

Comment:
use `parseYYYYmmddFormat` to avoid UTC issues

Reply:
ok I did some testing and both of our functions do the same thing lol